src/createjs/utils/BrowserDetect.js

```diff
@@ -63,7 +63,7 @@ this.createjs = this.createjs||{};
 	 */
 	function BrowserDetect() {
 		throw "BrowserDetect cannot be instantiated";
-	};
```

Comment:
No problem. I looked at all the semicolon changes, and I think they are correct, but if you see something that isn't just mention it no problem.